build-logic/build-platform/build.gradle.kts

```diff
@@ -73,5 +73,45 @@ dependencies {
         }
         api("net.bytebuddy:byte-buddy") { version { strictly("1.10.21") } }
         api("org.objenesis:objenesis") { version { strictly("3.1") } }
+        api("com.github.mwiede:jsch:0.2.0") {
```

Comment:
I do no think this is needed. This is because `jsch` is not a direct dependency for the build logic. Though it comes transitively from `jgit`

Reply:
I'm not sure if these files count as 'build logic', but jsch is added as a dependency here:

* https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/internal-performance-testing/build.gradle.kts#L61
* https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/version-control/build.gradle.kts#L22
* https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/version-control/build.gradle.kts#L35
* https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/resources-sftp/build.gradle.kts#L15
* https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/internal-integ-testing/build.gradle.kts#L90


And it's being used directly in code https://github.com/gradle/gradle/blob/36cad0c3425102540db84b80ec85702f69b948a0/subprojects/version-control/src/main/java/org/gradle/vcs/git/internal/GitVersionControlSystem.java#L20